staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4298,6 +4298,15 @@ type PodStatus struct {
 	// +featureGate=DynamicResourceAllocation
 	// +optional
 	ResourceClaimStatuses []PodResourceClaimStatus `json:"resourceClaimStatuses,omitempty" patchStrategy:"merge,retainKeys" patchMergeKey:"name" protobuf:"bytes,15,rep,name=resourceClaimStatuses"`
+
+	// TotalRequestedResources is the effective resource requests for this pod taking into consideration init
+	// and regular containers, and the pod overhead. If a resize of pod resources has been requested for a running
+	// pod, this field reflects the total desired resources for that pod which differs from actual resources
+	// allocated for the pod by the node.  If there is no effective resource request due to zero pod overhead and
+	// no requests on init or regular containers, the field is omitted.
+	// More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+	// +optional
+	TotalRequestedResources ResourceList `json:"totalRequestedResources,omitempty" protobuf:"bytes,17,opt,name=totalRequestedResources"`
```

Comment:
`Total` is a bit redundant.  `pod.status.requestedResources` should already imply pod scope.  Unless we have a very good argument, we should call it `requestedResources`.